release-1.6/release-notes-draft.md

```diff
@@ -59,12 +59,63 @@
 
 ## Cluster Lifecycle
 ### New Features
+- kubeadm is a simple tool for creating Kubernetes clusters on existing
```

Comment:
UPD. The features, landed under SIG-ClusterLifecycle (per spreadsheet and the features repo) are:

#### New Features
- **[alpha]** New Bootstrap Token authentication and management method. Works well with kubeadm.
- **[alpha]** Adds a new cloud-controller-manager binary that may be used for testing the new out-of-core cloudprovider flow
- **[beta]** Introduces an API for the Kubelet to request TLS certificates from the API server.
- **[beta]** kubeadm is enhanced and improved with a baseline feature set and command line flags that are now marked as beta.